tests/test_billboard.py

```diff
@@ -0,0 +1,413 @@
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "C:5",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "Eb:5",
+        "C:5",
+        "C:5",
+        "Eb:5",
+        "F:5",
+        "G:5",
+        "Bb:5",
+        "C:5",
+        "F#:5",
+        "F#:5",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "F:7",
+        "Bb:7",
+        "Bb:7",
+        "Eb:5",
+        "D:5",
+        "C:5",
+        "Bb:5",
+        "Ab:5",
+        "G:5",
+        "F:5",
+        "Eb:5",
+        "Bb:7",
+        "Bb:7",
+        "Eb:5",
+        "D:5",
+        "C:5",
+        "Bb:5",
+        "Ab:5",
+        "G:5",
+        "F:5",
+        "Eb:5",
+        "F:5",
+        "G:5",
+        "Ab:5",
+        "C:5",
+        "Db:5",
+        "Bb:5",
+        "Ab:7",
+        "A:5",
+        "Bb:5",
+        "B:5",
+        "C:5",
+        "C:5",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:5",
+        "C:5",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:7(#9)",
+        "C:1/1",
+        "C:1/1",
+        "C:5",
+        "F:5",
+        "Bb:5",
+        "Eb:5",
+        "Ab:5",
+        "Db:5",
+        "G:5",
+        "C:5",
+        "Eb:5",
+        "F#:5",
+        "A:5",
+        "C:7",
+        "C:7",
+        "N",
+        "N",
+        "N",
+        "N",
+    ]
+
+    # load a file which doesn't exist
+    chord_data_none = billboard._load_chords("fake/file/path")
+    assert chord_data_none is None
+
+    # load none
+    chord_data_none2 = billboard._load_chords("asdf/asdf")
```

Comment:
I think this tests the same case as above - what if you replace the path with `None`?